widgets/tooltip_list.py

```diff
@@ -75,7 +76,7 @@
 }
 
 camdata = {
-    "Camera Duration": "In frames, how long the camera should display for. Set to 0 if not a Grand Prix camera",
+    "Camera Duration": "In frames, how long the Grand Prix camera should display for. Total between all cams should be ~600. Set to 0 if not a Grand Prix camera",
```

Comment:
Nit: I've noticed that some of these sentences do not end with a `.`. To be correct, sentences need to end with proper punctuation in English.

Reply:
Should I update every tooltip in `tooltip_list` and individual `.json` files to end with a period?